docs/howto/distributed-tracing/src/main/resources/project-defaults.yml

```diff
@@ -0,0 +1,5 @@
+swarm:
+  jaeger:
+    service-name: my-traced-service
```

Comment:
It's better to name it with some business name or thorntail-example

Reply:
Ayea, `greeter` would definitely be better.

Reply:
Fixed.